mgmt_cli_test.py

```diff
@@ -1550,3 +1555,101 @@ def test_restore_data_without_manager(self):
                                "Please provide the 'mgmt_reuse_backup_snapshot_name' parameter.")
 
         self.test_restore_from_precreated_backup(snapshot_name, restore_outside_manager=True)
+
+
+class ReportType(Enum):
+    READ = 1
+    BACKUP = 2
+
+
+class ManagerBackupRestoreConcurrentTests(ManagerTestFunctionsMixIn):
+    def report_to_argus(self, report_type: ReportType, data: dict, label: str):
+        if report_type == ReportType.READ:
+            table = ReadResult(sut_timestamp=mgmt.get_scylla_manager_tool(
+                manager_node=self.monitors.nodes[0]).sctool.client_version_timestamp)
+        elif report_type == ReportType.BACKUP:
+            table = BackupResult(sut_timestamp=mgmt.get_scylla_manager_tool(
+                manager_node=self.monitors.nodes[0]).sctool.client_version_timestamp)
+        else:
+            raise InvalidArgument("Unknown report type")
+
+        for key, value in data.items():
+            table.add_result(column=key, value=value, row=label, status=Status.UNSET)
+        submit_results_to_argus(self.test_config.argus_client(), table)
+
+    def create_backup_and_report(self, mgr_cluster, label: str):
+        task = mgr_cluster.create_backup_task(location_list=self.locations, rate_limit_list=["0"])
```

Comment:
In terms of running the backup as fast as possible (and with as much cluster pressure as possible), SM 3.4 introduced `--transfers` flag which controls the amount of parallel file transfers (2 by default).

When downloading files during restore, we got the best results when setting it to `2*node_shard_cnt`.
Not sure if the same value would be the best for upload, but increasing it from 2 could definitely help.

Reply:
We need a way to accurately and _repeatedly_ measure the same thing.
If "cluster pressure" means lots of reads from disk during backup - then yes, this is what we need.

Reply:
tried to increase it to 16 and then to 32 and I dont see any difference. Moreover, the second run of backup timeouts since for some unknown reason the backup proves become significantly slower even when the concurrent read is over long time ago

Reply:
I will keep track of it, once [the issue](https://github.com/scylladb/scylla-manager/issues/4125) is resolved I will try to rerun it with different `--transfers` settings